### PR TITLE
update model: 1 class

### PR DESCRIPTION
**Pull request by ygnn123**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

File: models/object_detection/model_templates/vehicle-detection/readme.md [hunk 1]
```diff
@@ -7,8 +7,7 @@ Models that are able to detect vehicles.
...
 | vehicle-detection-0202 | 3.28 | 1.83 | 36.9 | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/v2/vehicle-detection-0202-1.pth), [model template](./vehicle-detection-0202/template.yaml) | 4 |
-| vehicle-detection-0203 | 112.34 | 24.11 | 43.8 | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/v2/vehicle-detection-0203.pth), [model template](./vehicle-detection-0203/template.yaml) | 4 |
-| vehicle-detection-0204 | 190.32 | 34.15 | 47.3 | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/v2/vehicle-detection-0204.pth), [model template](./vehicle-detection-0204/template.yaml) | 4 |
+| vehicle-detection-0203 | 112.34 | 24.11 | 43.8 | [snapshot](https://download.01.org/opencv/openvino_training_extensions/models/object_detection/v3/vehicle-detection-0203.pth), [model template](./vehicle-detection-0203/template.yaml) | 4 |
```
Comment (Ilya-Krylov) [inline models/object_detection/model_templates/vehicle-detection/readme.md:10]: fix AP

**Comment by Ilya-Krylov** [comment]
@ygnn123 check tests

**Comment by ygnn123** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by ygnn123** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
@ygnn123 please resolve conflicts

**Comment by ygnn123** [comment]
Ok, sure...

**Comment by ygnn123** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
@ygnn123 Could you please resolve conflicts, we did significant changes related to updating mmdetection backend. Some changes are required on configuration file side.

**Comment by Ilya-Krylov** [comment]
@ygnn123 Sorry for inconvenience

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by ygnn123** [comment]
run tests

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**